README.md

```diff
@@ -30,6 +30,14 @@ To install client side dependencies:
 * `brew install yarn`
 * `yarn`
 
+Then set up rails:
+
+* `sudo gem install rails`
```

Comment:
`bundle install` doesn't install the gems?

Reply:
Yep, I ran that command but since I had never used rails before, I still had to put in this command in order for it to understand what rails is.

Reply:
Hmm, sounds fishy. Running `bundle install` should pull in everything from the gemfile.

You also mentioned that something about Gemfile.lock. If there are changes in there, we do want to commit them. Please add it!

Reply:
Yep those changes are apart of the files changed!